app/src/main/java/com/elmeradrianv/shesafe/adapters/EmergencyContactsCardAdapter.java

```diff
@@ -116,7 +116,7 @@ private void deleteEmergencyContact(EmergencyContacts contact) {
         }
 
         private void makeACall(Context context) {
-            context.startActivity(new Intent(Intent.ACTION_DIAL)
+            context.startActivity(new Intent(Intent.ACTION_CALL)
```

Comment:
What's the difference between Dial and call? (haha I actually don't know, this isn't a trick question)